test/tap/groups/mysql84-gr/pre-proxysql.bash

```
@@ -1,82 +1,61 @@
+# The infra (infra-dbdeployer-mysql84-gr) is started by the framework AFTER this hook.
+# This hook configures ProxySQL with GR-specific fallback hostgroups, users, and routing.
 #
 
-[[ $(mysql --skip-ssl-verify-server-cert -h 2>&1) =~ skip-ssl-verify-server-cert ]] || export SSLOPT=--skip-ssl-verify-server-cert
+PROXY_CONTAINER="proxysql.${INFRA_ID}"
+INFRA="${DEFAULT_MYSQL_INFRA:-infra-dbdeployer-mysql84-gr}"
+MYSQL_HOST="${MYSQL_PRIMARY_HOST:-dbdeployer1}.${INFRA}"
 
-INFRA=infra-$(basename $(dirname "$0") | sed 's/-g[0-9]//' | sed 's/_.*//')
+run_admin() {
+    docker exec ${PROXY_CONTAINER} mysql -uradmin -pradmin -h127.0.0.1 -P6032 -e "$1" 2>&1 | grep -vP 'mysql: .?Warning' || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)